### DISCUSSION TITLE
Menu DropDown, Hydration failed error with NextJS 13

### DISCUSSION THREAD

**Discussion by frunny** [tailwindlabs/headlessui#2062] [Help] [answered]
I'm having trouble with the HeadlessUI Dropdown.
The Dropdown will be called from the Navbar component.

+ Error: Hydration failed because the initial UI does not match what was rendered on the server.
+ Error: There was an error while hydrating. Because the error happened outside of a Suspense boundary, the entire root will switch to client  #rendering.

NextJS 13.0.6
React 18.2.0
HeadlessUI insiders 1.7.4

**Comment by frunny** [comment, marked as answer]
Please close this issue, I solved it. 
I made a little stupid mistake in my code.

**Reply by sabbir2609** [reply]
and what was that?
